test/test_eos_unit.cpp

```diff
@@ -509,6 +509,76 @@ SCENARIO("Ideal gas entropy", "[IdealGas][Entropy]") {
   }
 }
 
+// A non-standard pattern where we do a reduction
+class CheckPofRE {
+ public:
+  CheckPofRE(Real *P, Real *rho, Real *sie, int N) : P_(P), rho_(rho), sie_(sie), N_(N) {}
+#pragma nv_exec_check_disable
```

Comment:
do we need to only include these pragmas when +cuda?

Reply:
Yes they only need to be there when `+cuda` but I assume they don't do anything otherwise. Is there an easy way to add them only in that case? This basically to remove the `calling device function from __host__ __device__ function` warning, because `Evaluate` is decorated as `PORTABLE_INLINE_FUNCTION`, but in this case we know we're on host. Could potentially move those pragmas to inside the base class, as well, which might be cleaner.

Reply:
alternatively could differentiate `EvaluateHost` and `EvaluateDevice` but that seems kinda gross IMO

Reply:
https://github.com/lanl/singularity-eos/blob/main/singularity-eos/eos/eos_eospac.hpp#L47

Reply:
Done.

Reply:
And moved it to the decorations for `Evaluate` in the base class, rather than in the functor.